keel-plugin/src/main/kotlin/com/netflix/spinnaker/config/AnnealingConfiguration.kt

```diff
@@ -0,0 +1,25 @@
+package com.netflix.spinnaker.config
+
+import com.netflix.spinnaker.keel.annealing.ResourceActuator
+import com.netflix.spinnaker.keel.annealing.ResourceCheckQueue
+import com.netflix.spinnaker.keel.annealing.spring.ApplicationEventResourceCheckListener
+import com.netflix.spinnaker.keel.annealing.spring.ApplicationEventResourceCheckQueue
+import org.springframework.boot.autoconfigure.condition.ConditionalOnBean
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean
+import org.springframework.context.ApplicationEventPublisher
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class AnnealingConfiguration {
+
+  @Bean
+  @ConditionalOnMissingBean(ResourceCheckQueue::class)
```

Comment:
Only register the Spring-based `ResourceCheckQueue` if there isn't another one registered. So we'll fall back to this locally / until we get an SQS/whatever backed implementation.